requirements/doc.in

```diff
@@ -3,5 +3,5 @@
 -r test.txt               # Core and test dependencies so sphinx-apidoc can import the code to document
 
 readme_renderer           # Validates README.rst for usage on PyPI
-Sphinx                    # Documentation builder
+Sphinx<2.0.0              # Documentation builder, 2.0.0 drops support for python 2.7
```

Comment:
Is this a good time to introduce a `constraints.txt` file?  Example: https://github.com/edx/edx-platform/commit/75b051b336aa308f7b8735ab0ddbacaf0563654f#diff-294960731653f2fdb7b827504766ff5d

Reply:
@jmbowman I considered that, but it looked like in platform we had just a single shared constraints file, whereas here, I would want to make one for docs and one for test?  That seemed heavier weight than necessary but there is something to be said for consistency.  Have you thought about how you'd like to do this when you need to limit only specific `.in` files?x

Reply:
Done.